.pre-commit-config.yaml

```
@@ -1,28 +1,28 @@
 repos:
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.3.1
+    rev: v3.19.1
```

I see an [error](https://github.com/euphorie/Euphorie/actions/runs/12828158903/job/35771605319?pr=785#step:7:77) that I think is caused by this upgrade.  This pyupgrade version pulls in a too new tokenize-rt.

Locally in a clean 3.8 venv, when installing pyupgrade I get `pyupgrade 3.16.0` and `tokenize-rt 6.0.0`.

```suggestion
    rev: v3.16.0
```

---

Thanks! I will revisit this one after https://github.com/euphorie/Euphorie/pull/786 is merged